src/lib/track-changes.ts

```diff
@@ -0,0 +1,165 @@
+/*!
```

Comment:
A lot of these functions are just to get displayable text for inspector. And they need to be grouped together based on this trait and not just in track-changes. For example getContributorTextContent doesn't have anything to do with track changes, but just creates displayable text for the node to be shown in the inspector. As I mentioned earlier you can create a single function that does that. Or as an alternative you can create a class that handles creation of displayable text for different types of nodes.